Weather App/viewModel/Storage.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+public protocol Storage {
```

Comment:
Could you add description to protocol and functions?

Reply:
👍